client/auth/request-login-email-form.jsx

```diff
@@ -0,0 +1,152 @@
+
+class RequestLoginEmailForm extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			hasSubmitted: false,
+			isEmailValid: false,
+		};
+	}
+	componentWillMount() {
+		const stateSetter = this.setFormState.bind( this );
+
+		this.formStateController = new formState.Controller( {
+			fieldNames: [ 'emailAddress' ],
+			debounceWait: 100,
+			validatorFunction: this.validate.bind( this ),
+			onNewState: stateSetter,
+			hideFieldErrorsOnChange: false,
+			initialState: {
+				emailAddress: {
+					value: '',
+				}
+			}
+		} );
+
+		stateSetter( this.formStateController.getInitialState() );
+	}
+
+	setFormState( state ) {
+		debug( 'setFormState', state );
+		this.setState( { form: state } );
+	}
+
+	handleChangeEvent( event ) {
+		this.formStateController.handleFieldChange( {
+			name: event.target.name,
+			value: event.target.value
+		} );
+	}
+
+	handleSubmit( event ) {
+		event.preventDefault();
+
+		const emailAddress = formState.getFieldValue( this.state.form, 'emailAddress' );
+		debug( 'form submitted!', emailAddress );
+
+		if ( ! emailAddress ) {
+			return;
+		}
+
+		this.setState( {
+			hasSubmitted: true,
+		} );
+
+		wpcom.undocumented().requestMagicLoginEmail( {
+			email: emailAddress
+		}, ( err, data ) => {
+			if ( err ) {
+				// @TODO handle error
+				return;
+			}
+			debug( 'Requeset successful', data );
+			window.location.replace( '/login/link-was-sent?email=' + encodeURIComponent( emailAddress ) );
+		} );
+	}
+
+	validate( formValues ) {
+		const emailAddress = formValues.emailAddress;
+		const isEmailValid = emailValidator.validate( emailAddress );
+		debug( 'validate', emailAddress, isEmailValid );
+		this.setState( {
+			isEmailValid
+		} );
+	}
+
+	render() {
+		const { currentUser, translate } = this.props;
+		return (
+			<div>
+				<LoggedOutForm onSubmit={ e => this.handleSubmit( e ) }>
+					<p>{
+						translate( 'Get a link sent to the email address associated ' +
+							'with your account to log in instantly without your password.' )
+					}</p>
+					{ currentUser && currentUser.username
+						? <p>{
+							translate( 'NOTE: You are already logged in as user: %(user)s', {
+								args: {
+									user: currentUser.username
+								}
+							} ) }</p>
+						: null }
+					<FormFieldset>
+						<FormTextInput
+							autoCapitalize="off"
+							autoFocus="true"
+							name="emailAddress"
+							placeholder="Email address"
+							value={ this.state.form.emailAddress.value }
+							onChange={ e => this.handleChangeEvent( e ) }
+						/>
+
+						<LoggedOutFormFooter>
+							<FormButton primary disabled={ this.state.hasSubmitted || ! this.state.isEmailValid }>
+								{ translate( 'Request Email' ) }
+							</FormButton>
+						</LoggedOutFormFooter>
+					</FormFieldset>
+				</LoggedOutForm>
+				<LoggedOutFormLinks>
+					<LoggedOutFormLinkItem href={ config( 'login_url' ) }>
+						{ translate( 'Enter a password instead' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Enter a Password' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).